src/react/components/Card.js

```diff
@@ -21,12 +21,14 @@ CardError.propTypes = {
   message: PropTypes.string
 };
 
-const Card = ({ children, title, loading, error, className }) => {
+const Card = ({ children, title, loading, error, className, secondaryContent }) => {
   const classNames = ['Card', className].join(' ');
+
   return (
     <div className={classNames}>
       <div className="Card__title">
         <h2>{title}</h2>
+        {secondaryContent && secondaryContent}
```

Comment:
@developish is this intentional?  how is it different than `{secondaryContent}'?